.github/workflows/suggest-review-fixes.yml

```diff
@@ -62,7 +62,10 @@ jobs:
             <only the code you are proposing as the replacement for the selected lines>
             \`\`\`
 
-            For suggestions where the original code spans multiple lines, you must also include a \`start_line\` field in the JSON comment representing the first line of the range covered by the suggestion; \`line\` should be the last line of the range.
+            As an example:
+            {
+              "12345678": "Updated the comment to address feedback, ```suggestion\n// New version of the comment here\n```"
```

Comment:
On the off chance the model confuses code comments with review comments, maybe we should use something like `the print statement`?